sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -39,28 +39,28 @@ The way to obtain group relationship that will determine which graph api will be
 * **direct**: the default value, get groups that the user is a direct member of. For details, see [list memberOf][graph-api-list-member-of] api.
 * **transitive**: Get groups that the user is a member of, and will also return all groups the user is a nested member of. For details, see [list transitive memberOf][graph-api-list-transitive-member-of] api.
 
-### Web Server
-Based on Azure AD as a Web Server, it uses OAuth2 authorization code flow to authentication, and authorizes resources based on the scopes or roles claim in the access token. 
-Provide a convenient way to quickly access other resource server, other resources should be registered as `ClientRegistration`, use `@RegisteredOAuth2AuthorizedClient` annotation to mark the client resource, Security will help automatically obtain valid access tokens based on the root refresh token, business methods will use the corresponding access token to request client resources.
+### Web application
+Based on Azure AD as a Web application, it uses OAuth2 authorization code flow to authentication, and authorizes resources based on the scopes or roles claim in the access token. 
```

Comment:
I think we should replace `scopes` with `groups` here. And it is the same with below 2 graphs, update the flow of `Evaluate permission based on roles` to `Evaluate permission based on groups or roles`.

Reply:
Besides, I think in the first graph, we need to find a better way to diaplay the scenerios of access denied and data returned. Making them parallel is confusing because it may be read as two successive flows.